### PR TITLE
feat(editor): Add selection navigation using the keyboard on new canvas

### PR DESCRIPTION
**Pull request by alexgrozav**
## Summary

https://github.com/user-attachments/assets/826526ea-22da-4112-8615-83ed5f9b628e

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/CAT-322/change-arrow-keys-behaviour-to-be-consistent-with-old-canvas

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Comment by alexgrozav** [comment]
Blocked until https://github.com/bcakmakoglu/vue-flow/issues/1689 is solved.

File: packages/editor-ui/src/composables/useCanvasTraversal.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import type { CanvasNode } from '@/types';
```
Comment (elsmr) [inline packages/editor-ui/src/composables/useCanvasTraversal.ts:1]: 👏

File: packages/editor-ui/src/components/canvas/Canvas.test.ts [hunk 2]
```diff
@@ -119,13 +119,6 @@ describe('Canvas', () => {
 				[
 					{
 						id: '1',
-						type: 'position',
-						dragging: true,
-						from: {
-							x: 100,
-							y: 100,
-							z: 0,
-						},
```
Comment (elsmr) [inline packages/editor-ui/src/components/canvas/Canvas.test.ts:128]: is this expected?

**Review by elsmr — APPROVED** [review]
LGTM 🎉  Compared with the old canvas and it works the same, only much faster! 🏃

File: packages/editor-ui/src/components/canvas/Canvas.test.ts (hunk 2 above)
Comment (alexgrozav) [reply packages/editor-ui/src/components/canvas/Canvas.test.ts:128]: Yup, position listener got changed.

**Comment by janober** [comment]
Got released with [`n8n@1.69.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.69.0)

**Pull request merged**